src/src/hooks/Topics.js

```diff
@@ -37,29 +37,24 @@ export function useTopics() {
 }
 
 export function useDeleteTopic() {
-  const { triggerError, sendRequest } = useSelfServiceRequest();
+  const { triggerErrorWithTitleAndDetails, sendRequest } =
```

Comment:
It is a little funny that our errorToast management is exposed by our self-service request instead of `useError`.
It is a minor thing, I guess 🤷